internal/entities/treecluster.go

```diff
@@ -36,4 +36,5 @@ type TreeCluster struct {
 	Longitude      float64
 	Trees          []*Tree
 	SoilCondition  TreeSoilCondition
+	Name           *string
```

Comment:
I don't think I would use a pointer here

```suggestion
	Name           string
```